test/e2e/testdata/helm-repo/local/index.yaml

```diff
@@ -5,3 +5,4 @@ entries:
       urls:
         - http://127.0.0.1:9080/argo-e2e/testdata.git/helm-repo/helm-1.0.0.tgz
       version: 1.0.0
+      name: helm
```

Comment:
QQ: Is the name field required here? I do not think this change is related to `skipSchemaValidation` flag changes.

Reply:
It is. In helm 3.16, this property is now required (its not documented anywhere, but the test errors were saying it was a missing required attribute). Given that helm 3.16 is a requirement for this feature, it was added here